### ISSUE TITLE
Search Results - Non Text Contrast Issues

### ISSUE THREAD

**Issue by ewlarson** [geobtaa/geoportal#521]
1. SC 1.4.11 AA – Non Text Contrast – “Search Button” – focus indicators lack 3:1 contrast ratio with
adjacent background (Focus indicator: #016EA7, Adjacent Background: #023C5A, contrast ratio:
2.1:1)
2. SC 1.4.11 AA – Non Text Contrast – “Advanced Link, search field” – focus indicators lack 3:1
contrast ratio with adjacent background (Focus indicator: #203558, Adjacent Background: #393939,
contrast ratio: 1.1:1)
3. SC 1.4.11 AA – Non Text Contrast – “Remove (X) links” – focus indicators lack 3:1 contrast ratio with
adjacent background (Focus indicator: #B6BABD, Adjacent Background: #6C757C, contrast ratio:
2.4:1)
